pkg/providers/cloudstack/apibuilder.go

```diff
@@ -0,0 +1,119 @@
+package cloudstack
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cloudstackv1 "sigs.k8s.io/cluster-api-provider-cloudstack/api/v1beta2"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/clusterapi"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	anywhereTypes "github.com/aws/eks-anywhere/pkg/types"
+)
+
+const (
+	CloudStackMachineTemplateKind      = "CloudStackMachineTemplate"
+	CloudStackInfrastructureAPIVersion = "infrastructure.cluster.x-k8s.io/v1beta2"
```

Comment:
Can we use the `InfrastructureAPIVersion` here from `cloudstackv1` or is there no method that gets this specific version?

Reply:
Indeed we can, thanks